manifest.json

```diff
@@ -10,9 +10,10 @@
                 "client/js/appConstants.js",
                 "client/js/apiKeys.js",
                 "client/js/apiService.js",
+                "client/js/modalContent.js",
                 "client/js/content.js"
             ],
-            "matches": ["https://www.netflix.com/browse"]
+            "matches": ["https://www.netflix.com/*"]
```

Comment:
This will intefere with other webpages like /watch /search etc
Make it /browse*